r4/fhir_audit_handler.bal

```diff
@@ -79,15 +89,25 @@ public isolated function handleAuditEvent(AuditConfig auditConfig, FHIRContext f
         entityWhatReference: fhirContext.getRawPath()
     };
 
-    http:Client|http:ClientError auditClient = new (auditConfig.auditServiceUrl);
+    http:Client|http:ClientError auditClient = new (auditConfig.auditServiceUrl, retryConfig = {
+        interval: 5,
+        count: 3,
+        backOffFactor: 2.0,
+        maxWaitInterval: 30
```

Comment:
Are we panning to make these configurables?

Reply:
Since it is a server to server communication and as a user, a FHIR API developer has no idea what are the best values for these - I decided against that. But we can make them configurable later, if a need arise.